Documentattion for Airios integration (#37924)
home-assistant/home-assistant.github.io: source/_integrations/airios.markdown

---
title: Airios
description: Instructions on how to integrate Airios ventilation units into Home Assistant.
ha_category:
  - Climate
  - Fan
ha_config_flow: true
ha_release: 2025.4
ha_iot_class: Local Polling
ha_domain: airios
ha_platforms:
  - fan
ha_codeowners:
  - '@scabrero'
ha_integration_type: hub
---

The **Airios** {% term integration %} allows you to control and monitor ventilation units and accessories from different manufacturers.

**Airios** develops and produces components for residential ventilation systems that final manufacturers use to build their products upon, from controller boards to remote controls or sensors. These components communicate over a proprietary RF protocol from Honeywell called Ramses II in the 868Mhz band.

> 

Reviewer: ```suggestion

This {% term integration %} has been tested with the following manufacturers.

```

> An RF bridge is needed for Home Assistant to access the RF network. There are two bridge models with different interfaces. The **Airios** BRDG-02R13 has a RS485 serial interface (Modbus-RTU) and the BRDG-02EM23 is an Ethernet device (Modbus-TCP).
> 
> {% warning %}
> Only the RS485 serial bridge is supported by this {% term integration %}.
> {% endwarning %}

Reviewer: ```suggestion
## Prerequisites

An RF bridge is needed for Home Assistant to access the RF network. There are two bridge models with different interfaces. The **Airios** BRDG-02R13 has a RS485 serial interface (Modbus-RTU) and the BRDG-02EM23 is an Ethernet device (Modbus-TCP).

- Before you can use this integration, make sure you have the **Airios** BRDG-02R13 with the RS485 serial bridge. The BRDG-02EM23 Ethernet device is not supported.
```

> This {% term integration %} has been tested with the following manufacturers.

Reviewer: ```suggestion
```

---
title: Airios
description: Instructions on how to integrate Airios ventilation units into Home Assistant.
ha_category:
  - Climate
  - Fan
ha_config_flow: true
ha_release: 2025.4
ha_iot_class: Local Polling
ha_domain: airios
ha_platforms:
  - fan
ha_codeowners:
  - '@scabrero'
ha_integration_type: hub
---

The **Airios** {% term integration %} allows you to control and monitor ventilation units and accessories from different manufacturers.

**Airios** develops and produces components for residential ventilation systems that final manufacturers use to build their products upon, from controller boards to remote controls or sensors. These components communicate over a proprietary RF protocol from Honeywell called Ramses II in the 868Mhz band.

> This {% term integration %} has been tested with the following manufacturers.

Reviewer: will you provide a list here? Otherwise I would remove this statement.

Author: Hi, thanks for review. Yes, the idea is to provide a list where each entry is a link to its virtual integration page. And in each virtual integration page list the compatible/tested models and accessories.

Reviewer: aha, ok. thank you for clarifying.

## Prerequisites

An RF bridge is needed for Home Assistant to access the RF network. There are two bridge models with different interfaces. The **Airios** BRDG-02R13 has a RS485 serial interface (Modbus-RTU) and the BRDG-02EM23 is an Ethernet device (Modbus-TCP).

- Before you can use this integration, make sure you have the **Airios** BRDG-02R13 with the RS485 serial bridge. The BRDG-02EM23 Ethernet device is not supported.

{% include integrations/config_flow.md %}

> ## Platforms

Reviewer: ```suggestion
## Supported functionality
```

ha_category:
  - Climate
  - Fan
ha_config_flow: true
ha_release: 2025.4
ha_iot_class: Local Polling
ha_domain: airios
ha_platforms:
  - fan
ha_codeowners:
  - '@scabrero'
ha_integration_type: hub
---

The **Airios** {% term integration %} allows you to control and monitor ventilation units and accessories from different manufacturers.

**Airios** develops and produces components for residential ventilation systems that final manufacturers use to build their products upon, from controller boards to remote controls or sensors. These components communicate over a proprietary RF protocol from Honeywell called Ramses II in the 868Mhz band.

This {% term integration %} has been tested with the following manufacturers.

## Prerequisites

An RF bridge is needed for Home Assistant to access the RF network. There are two bridge models with different interfaces. The **Airios** BRDG-02R13 has a RS485 serial interface (Modbus-RTU) and the BRDG-02EM23 is an Ethernet device (Modbus-TCP).

- Before you can use this integration, make sure you have the **Airios** BRDG-02R13 with the RS485 serial bridge. The BRDG-02EM23 Ethernet device is not supported.

> 

Reviewer: ```suggestion
```
